partner_academic_title/models/res_partner.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def _get_separator(self):
+        return ', '
+
+    @api.depends('academic_title_ids', 'academic_title_ids.sequence')
+    @api.one
```

Comment:
@api.one is deprecated. Use self.ensure_one() as first line in method.

Reply:
Changes done, but ensure_one() causes an error when building the runbot instance because the method is applied on a set of records. Maybe iterating with something like "for rec in self" inside the function?

Reply:
Yes that would be the solution. I had not taken into account the possibility that this could be called for multiple records. ```for rec in self``` or ```for this in self``` as i prefer (but that is purely personal taste), is fine.